pkg/params/settings/config.go

```diff
@@ -231,7 +235,9 @@ func ConfigToSettings(logger *zap.SugaredLogger, setting *Settings, config map[s
 		logger.Infof("CONFIG: hub catalog name set to %v", config[HubCatalogNameKey])
 		s.Name = config[HubCatalogNameKey]
 	}
+	mu.Lock()
 	setting.HubCatalogs["default"] = s
```

Comment:
this block should be removed

Reply:
True that complete block is not needed as its repeated